R/scale-continuous.R

```diff
@@ -105,12 +110,17 @@ scale_y_continuous <- function(name = waiver(), breaks = waiver(),
                                na.value = NA_real_, trans = "identity",
                                guide = waiver(), position = "left",
                                sec.axis = waiver()) {
+  call <- caller_call()
+  if (is.null(call) || !startsWith(as.character(call[[1]]), "scale_")) {
```

Comment:
I think this change created a problem when calling `scale_y_continous` within another package function via the double colon.

`as.character(call[[1]])` contents is of length 3 and the if clause fails

```R
# [1] "::"         "tern"       "g_lineplot"
```

Reply:
My guess is that `caller_call` is catching the parent call instead of the `scale_*` name